src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -171,23 +171,41 @@ salmon_rnaseq_bulk:
     primary: false
     vitessce-hints: []
 
-SNAREseq:
-    description: SNARE-seq
+SNARE2-ATACseq:
+    description: snATACseq (SNARE-seq2)
+    alt-names: [SNAREseq]
+    primary: true
+    vitessce-hints: []
+
+SNARE2-RNAseq:
+    description: snRNAseq (SNARE-seq2)
```

Comment:
```suggestion
    description: snRNAseq (SNARE2-seq)
```

Reply:
Drat- @pdblood was explicit about it being SNARE-seq2 rather than SNARE2-seq.  I'm not sure why but we probably have to block until he comes back and can clarify this.

Reply:
You can reverse to SNARE-seq2, as long as we are consistent.

Reply:
@ngehlenborg: I don't see `SNARE2-seq` in the original version of this. Were you referring to the keys `SNARE2-ATACseq` and `SNARE2-RNAseq`. If we change these, are you suggesting `SNARE-ATACseq2` and `SNARE-RNAseq2`?

Reply:
@ngehlenborg @jswelling : I confirmed with UCSD that the description should remain `SNARE-seq2 `. Nils, assuming you were referring to the keys above: are those something that a user will ever see?

Reply:
@pdblood @ngehlenborg I believe we're agreed upon the following.  If I don't hear otherwise I'll set these up, setting the old keys as 'alt-names' so they will be translated properly.

key: SNARE-ATACseq2  description: snATACseq (SNARE-seq2)

key: SNARE-RNAseq2  description: snRNAseq (SNARE-seq2)

Reply:
@jswelling looks good to me. @ngehlenborg can you approve?

Reply:
As far as I can tell, these changes make everything consistent and that is all I was looking for.